styles/_grid.scss

```diff
@@ -0,0 +1,25 @@
+.outer-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+}
+
+.container {
+
+  width: 75%;
```

Comment:
This should use the `max-content-width` variable to center up the content on the page and set the `max-width` so that it doesn't limit the size of things on smaller screens.